jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EjbSchedulerService.java

```diff
@@ -131,15 +133,16 @@ public boolean isValid(GlobalJobHandle jobHandle) {
         return true;	    
 	}
 	
-	private String getJobName(JobContext ctx, Long id) {
+    protected String getJobName(JobContext ctx, Long id) {
 
         String jobname = null;
         
         if (ctx instanceof ProcessJobContext) {
             ProcessJobContext processCtx = (ProcessJobContext) ctx;
-            jobname = processCtx.getSessionId() + "-" + processCtx.getProcessInstanceId() + "-" + processCtx.getTimer().getId();
+            String jobNamePrefix = processCtx.getSessionId() + "-";
```

Comment:
maybe we could use the actual deployment id as the prefix?

```
String jobNamePrefix = processCtx.getKnowledgeRuntime().getEnvironment().get(EnvironmentName.DEPLOYMENT_ID) + "-";
```

Reply:
@mswiderski tried that but it is force by the transitive dependency not to use kie api